runtime-image/jdk11-xp-overrides.yaml

```diff
@@ -0,0 +1,28 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp10-openjdk11-runtime-openshift-rhel8"
```

Comment:
No runtime image for xp is expected, we would use 7.3.1 one.

Reply:
@jfdenise Ken is right, we will deliver a runtime image for EAP XP (distinct from the 7.3.1 runtime image).

(they are technically equivalent but have different lifecycle / support policy)

Reply:
@jmesnil , ok I missed this. Thanks.